### PR TITLE
Simple chart design updates

### PR DESCRIPTION
**Pull request by wpears**
Basket of simple-chart design updates from super-reg#276

### CODE REVIEW

File: cfgov/unprocessed/js/routes/on-demand/simple-chart/common-styles.js [hunk 1]
```diff
@@ -14,7 +15,20 @@ const styles = {
...
+  legend: {
+    enabled: false,
+    symbolWidth: 45,
+    floating: true,
+    layout: 'vertical',
+    align: 'right',
+    verticalAlign: 'top',
+    itemMarginBottom: 4,
+    itemStyle: {
+      color: '#5a5d61',
```
Comment (anselmbradford) [inline cfgov/unprocessed/js/routes/on-demand/simple-chart/common-styles.js:27]: `color: @gray` ?

**Comment by anselmbradford** [comment]
Where is simple charted used?

File: cfgov/unprocessed/js/routes/on-demand/simple-chart/datetime-styles.js [hunk 2]
```diff
@@ -1,45 +1,40 @@
...
   },
   xAxis: {
     ...styles.xAxis,
     type: 'datetime',
-    minRange: 30 * 24 * 3600 * 1000,
     startOnTick: true,
     labels: {
-      ...styles.xAxis.labels,
-      format: 'Jan<br/>{value:%Y}'
+      ...styles.xAxis.labels
```
Comment (anselmbradford) [inline cfgov/unprocessed/js/routes/on-demand/simple-chart/datetime-styles.js:18]: If IE11 support is needed, you might double-check transpiling of the spread operator in IE11. You may need to include a polyfill.

Comment (wpears) [reply cfgov/unprocessed/js/routes/on-demand/simple-chart/datetime-styles.js:18]: The transpiling appears to work in my testing

**Comment by wpears** [comment]
@ans Simple-chart is currently what powers the ["Enforcement by the Numbers" page](https://www.consumerfinance.gov/enforcement/enforcement-by-the-numbers/) and there are now some beta charts that this PR is supporting at https://beta.consumerfinance.gov/data-research/chart-test/precipitation-multi/ (and siblings)

File: cfgov/unprocessed/js/routes/on-demand/simple-chart/datetime-styles.js [hunk 3]
```diff
@@ -57,20 +52,20 @@ const datetime = {
...
+      type: 'millisecond',
+      count: 3 * msYear,
       text: '3y',
       title: 'View 3 year'
     },
     {
-      type: 'year',
-      count: 5,
+      type: 'millisecond',
+      count: ( 5 * msYear ) + 86400000,
```
[the remark this answers is not in the document]
Comment (wpears) [reply cfgov/unprocessed/js/routes/on-demand/simple-chart/datetime-styles.js:68]: That's the number of milliseconds in a day. For some reasons (highcharts is buggy af) for the 5 year button I had to introduce this 1 day offset to get it working correctly.

[the remark this answers is not in the document]
Comment (schbetsy) [reply cfgov/unprocessed/js/routes/on-demand/simple-chart/datetime-styles.js:68]: A day in milliseconds, I assume? Why does the 5-year span need that offset but not 3-year?

**Review by schbetsy — APPROVED** [review]

File: cfgov/unprocessed/js/routes/on-demand/simple-chart/datetime-styles.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (wpears) [reply cfgov/unprocessed/js/routes/on-demand/simple-chart/datetime-styles.js:68]: GREAT QUESTION

**Pull request merged**